hygen/_templates/source/new/source.ejs.t

```diff
@@ -11,7 +11,7 @@ const source = {
 <% if (platform === 'carto-cloud-native') { -%>
   connection: '<%- connection %>',
 <% } -%>
-  data: '<%- data -%>',
+  data: `<%- data -%>`,
```

Comment:
I think we can still have an error if the user enters backticks in the query specified in the code generator. Could we escape the characters?

Reply:
We can `.replace(/`/g, '``');`